airflow/providers/amazon/aws/secrets/secrets_manager.py

```diff
@@ -60,15 +59,23 @@ class SecretsManagerBackend(BaseSecretsBackend, LoggingMixin):
 
     def __init__(
         self,
-        connections_prefix: str = 'airflow/connections',
-        variables_prefix: str = 'airflow/variables',
-        profile_name: Optional[str] = None,
-        sep: str = "/",
+        connections_prefix=None,
```

Comment:
Why do these need to be None? this controls where in the secrets hierarchy to look at, so should be a required field, no?

Reply:
That was because I was not able to make it works using connections prefix, so I ended up ignoring them. In my set up those values are None and I just type the entire name of the secret.

Reply:
The following should work in your case (can you try):

```ini
[secrets]
backend = airflow.contrib.secrets.aws_secrets_manager.SecretsManagerBackend
backend_kwargs = {"connections_prefix": "", "sep": ""}
```